### ISSUE TITLE
Renderer jako moduł

### ISSUE THREAD

**Issue by BonusPlay3** [BonusPlay3/The-Gruntmaster-6000#1]
Seems like a bad design, since then renderer has to implement all the object types render functions.

**Comment by szymonszl** [comment]
No, the renderer will be to engine "transparent", engine will just run "render()", and renderer would read all needed data from a set of global vars/tables.
